src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -502,7 +502,7 @@ protected function initializeBundles()
      */
     protected function getContainerClass()
     {
-        return $this->name.ucfirst($this->environment).($this->debug ? 'Debug' : '').'ProjectContainer';
+        return preg_replace('/(^[0-9]*|[^a-zA-Z0-9_]+)/', '', $this->name.ucfirst($this->environment)).($this->debug ? 'Debug' : '').'ProjectContainer';
```

Comment:
thus, if we fix it here, we need to fix it for the other dumped classes (the url matcher and the url generator)